src/modules/messages/proxyPluralMessage.ts

```diff
@@ -28,7 +34,7 @@ export const proxyPluralMessage = async (
       webhooks.find((w) => w.owner && bot.user && w.owner.id === bot.user.id) ||
       (await channel.createWebhook({ name: "Melody's Plural System" }));
 
-    const content = {
+    const content: ProxyContent = {
```

Comment:
```suggestion
    const content: WebhookMessageCreateOptions = {
```

We can use the built-in typedef.